app/web/features/dashboard/MinimalUserProfileCard.tsx

```diff
@@ -1,41 +1,53 @@
-import { Box, Card, Link as MuiLink, Typography } from "@material-ui/core";
+import { Card, Link as MuiLink, Typography } from "@material-ui/core";
 import Avatar from "components/Avatar";
 import { DASHBOARD } from "i18n/namespaces";
 import Link from "next/link";
 import { useTranslation } from "next-i18next";
 import { User } from "proto/api_pb";
 import { routeToProfile } from "routes";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    display: "flex",
+    flexDirection: "row",
+    paddingTop: theme.spacing(1),
+    paddingBottom: theme.spacing(1),
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(2),
```

Comment:
```suggestion
    padding: theme.spacing(1, 2),
```
This is pretty minor, but I guess this shorthand would emulate the `px` and `py` props `Box` was trying to achieve with slightly less code.

Reply:
ah great, thanks!